### ISSUE TITLE
Should the Return Skater icon be changed for the conservative character set?

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/energy-skate-park#356]
I noticed that when I change to the conservative character set, the Intro Screen icon no longer has the girl in shorts. I was wondering if the Return Skater icon should be changed as well?
<img width="124" alt="Screenshot 2022-12-07 at 8 29 46 AM" src="https://user-images.githubusercontent.com/87318828/206216772-29281fc4-8a6f-4102-a221-132abc6ab9ad.png">

**Comment by arouinfar** [comment]
Good catch @Nancy-Salpepi! Let's use skater 1 on the button and swap it between sets.

**Comment by AgustinVallejo** [comment]
One question regarding this issue. The button is now changing based on regional preference, but should it also correspond to the selected skater? i.e. If I have the goat selected, should the button display a goat instead of the basic skater? I guess that'd be a different issue anyways

![image](https://user-images.githubusercontent.com/37052186/207117915-aecd62e9-ce8c-4fb6-9d64-584ba66ade0b.png)
